ComprefaceTestApp/DTOs/ExampleSubject/AddExampleSubject/AddExampleSubjectRequest.cs

```diff
@@ -1,14 +1,10 @@
-﻿using System.Text.Json.Serialization;
+﻿namespace ComprefaceTestApp.DTOs.ExampleSubject.AddExampleSubject;
 
-namespace ComprefaceTestApp.DTOs.ExampleSubject.AddExampleSubject;
-
-public class AddExampleSubjectRequest
+public class AddExampleSubjectRequest : SubjectBase
 {
-    public string Subject { get; set; }
-    
     public decimal? DetProbThreShold { get; set; }
 
-    public string FilePath { get; set; }
+    public string? FilePath { get; set; }
 
```

Comment:
string is by default nullable so no need for `?`  here

Reply:
yeah, but...
![image](https://user-images.githubusercontent.com/39197664/209141251-8c71883c-b3ce-4abb-9a7e-c5a320f471a7.png)

Reply:
From C# 8.0 null-context definition has appeared. So that's why we need mark string like a nullable. Also we can set up it global in the project and disable this warnings.